src/core/cgroup.c

```diff
@@ -1302,8 +1335,10 @@ static void cgroup_context_apply(
         }
 
         if ((apply_mask & CGROUP_MASK_CPUSET) && !is_local_root) {
-                cgroup_apply_unified_cpuset(u, &c->cpuset_cpus, "cpuset.cpus");
-                cgroup_apply_unified_cpuset(u, &c->cpuset_mems, "cpuset.mems");
+                CPUSet *cpus = cgroup_context_allowed_cpus(c, state);
+                CPUSet *mems = cgroup_context_allowed_mems(c, state);
+                cgroup_apply_unified_cpuset(u, cpus, "cpuset.cpus");
```

Comment:
i'd just shorten this to:

```c
cgroup_apply_unified_cpuset(u, cgroup_context_allowed_cpus(c, state), "cpuset.cpus");
```

no need to do this in two steps, the expression doesn't become that complex by that yet

Reply:
Yep, done.